formats/ome-tiff/specification.txt

```diff
@@ -340,64 +340,66 @@ distributed across multiple files. Each of the files in the set has identical
 metadata apart from the `UUID`, the unique identifier in each file. For
 example the identifiers could be distributed as follows:
 
-**tubhiswt_C0_TP0.ome.tif** with ID 69bfd94f-2d95-41ae-887d-26e0c11fd606
+**tubhiswt_C0_TP0.ome.tif** with ID 45c8bf18-6aa2-478c-9080-e0b0d3eb1f70
 
 ::
 
-    <OME xmlns="http://www.openmicroscopy.org/Schemas/OME/2015-01"
+    <OME xmlns="http://www.openmicroscopy.org/Schemas/OME/2016-06"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         UUID="urn:uuid:69bfd94f-2d95-41ae-887d-26e0c11fd606"
-         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2015-01
-                             http://www.openmicroscopy.org/Schemas/OME/2015-01/ome.xsd">
+         Creator="OME Bio-Formats 5.2.0-m4"
+         UUID="urn:uuid:45c8bf18-6aa2-478c-9080-e0b0d3eb1f70"
+         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-06
+         http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd">
     ...
         <Pixels BigEndian="false" DimensionOrder="XYZTC" ID="Pixels:0"
                 Interleaved="false" SignificantBits="8" SizeC="2" SizeT="43"
                 SizeX="512" SizeY="512" SizeZ="10" Type="uint8">
     ...
           <TiffData FirstC="0" FirstT="0" FirstZ="0" IFD="0" PlaneCount="1">
             <UUID FileName="tubhiswt_C0_TP0.ome.tif">
-              urn:uuid:69bfd94f-2d95-41ae-887d-26e0c11fd606
+              urn:uuid:45c8bf18-6aa2-478c-9080-e0b0d3eb1f70
             </UUID>
           </TiffData>
     ...
-          <TiffData FirstC="0" FirstT="1" FirstZ="0" IFD="0" PlaneCount="1">
-            <UUID FileName="tubhiswt_C0_TP1.ome.tif">
-              urn:uuid:16093e77-8531-43c5-83df-c71efbd212d0
+          <TiffData FirstC="0" FirstT="0" FirstZ="1" IFD="1" PlaneCount="1">
+            <UUID FileName="tubhiswt_C0_TP0.ome.tif">
```

Comment:
Ah! I was just saying to @dgault in slack that it looked right when I doublechecked but yes, I could change the FirstT value too